ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml

```diff
@@ -15,13 +15,26 @@ images:
 promotion:
   name: "4.3"
   namespace: ocp
+raw_steps:
+- rpm_image_injection_step:
+    from: src
+    to: mco-rpms
 resources:
   '*':
     limits:
       memory: 8Gi
     requests:
       cpu: 100m
       memory: 200Mi
+rpm_build_commands: |-
```

Comment:
You'll need this on the fcos branch.

Reply:
(copy this from -master.yaml to -fcos.yaml once we have all review steps in place)